.github/workflows/code.yml

```diff
@@ -32,13 +32,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python: ["3.8", "3.9", "3.10", "3.11"]
-        install: ["-e .[dev]"]
-        # Make one version be non-editable to test both paths of version code
```

Comment:
You have removed the non-editable test which does pick up a few issues that editable tests don't.

To resolve this remove "3.9" from your matrix above and then add:
```python
      include:		
         - os: "ubuntu-latest"		
           python: "3.9"		
           install: ".[dev]"
```

Reply:
It's actually testing against the non-editable version of all 3 versions of Python we claim to support:

       os: ["ubuntu-latest"] # can add windows-latest, macos-latest
        python: ["3.9", "3.10", "3.11"]
        install: [".[dev]", "-e .[dev]"]

Runs against every combinationf of python, install[, os]
![image](https://github.com/DiamondLightSource/dls-bluesky-core/assets/53935796/79844312-85c1-4125-8882-838f75c140fc)